lumigator/frontend/src/App.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+	<div id="app">
+		<div class="header"></div>
+		<div class="l-menu-container">
+			<l-menu />
+		</div>
+		<div class="l-main-container">
+			<router-view v-slot="{ Component }">
+				<transition name="transition-fade">
+					<component :is="Component" @s-disable-scroll="disableScroll = $event" />
+				</transition>
+			</router-view>
+		</div>
+	</div>
+</template>
+<script setup>
+import LMenu from '@/components/LMenu.vue'
```

Comment:
I think this import is failing, should it be updated to `/components/organisms/LMenu.vue` ?